config/locales/en.yml

```
@@ -30,4 +30,63 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  presenters:
+    dashboard:
+      index_presenter:
+        links:
+          schedule:
+            title: "Schedule"
+            text: "Schedule visits for patients. Field workers like nurses, doctors, and physiotherapists can see patients who need care in the next week and schedule their visits."
+            icon: "icon_outline_clock"
+          visit:
+            title: "Visit"
+            text: "Visit a patient and record their details."
+            icon: "icon_outline_clock"
+          lsg:
+            title: "LSG"
+            text: "Visit a patient and record their details."
```

The text should be changed 
```suggestion
            text: "View LSG data"
```